app/views/layouts/application.html.erb.tt

```diff
@@ -25,6 +25,11 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+    <%%# Prevents the app is indexed by Google %>
+    <%% if ENV.fetch("NO_INDEX", "YES") == "YES" %>
+      <meta name="robots" content="noindex" />
+    <%% end %>
```

Comment:
Typically ERB tags are `<%` not `<%%` - I can't find any docs about `<%%` - any chance it's a typo? Or am I missing something?

Reply:
This is template file, not erb.  It's an escape sequence for .tt files.